### PR TITLE
Reorder imports with isort

### PR DESCRIPTION
**Pull request by yhn112**

### CODE REVIEW

**Comment by borzunov** [comment]
Please consider https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

File: .github/workflows/check-style.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          options: "--check --diff"
+          version: "21.6b0"
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: isort/isort-action@master
```
Comment (borzunov) [inline .github/workflows/check-style.yml:21]: nit: \n

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 [tool.black]
 line-length = 119
 required-version = "21.6b0"
+[tool.isort]
+profile = "black"
+line_length = 119
```
Comment (borzunov) [inline pyproject.toml:6]: nit: \n

**Review by borzunov — APPROVED** [review]

File: examples/albert/run_trainer.py [hunk 3]
```diff
@@ -8,21 +8,20 @@
 
 import torch
 import transformers
+import utils
+from arguments import AlbertTrainingArguments, AveragerArguments, CollaborationArguments, DatasetArguments
```
Comment (mryab) [inline examples/albert/run_trainer.py:12]: Imports of local modules should not get mixed with third-party libraries. Consider configuring options such as [known-first-party](https://pycqa.github.io/isort/docs/configuration/options.html#known-first-party) and [known-local-folder](https://pycqa.github.io/isort/docs/configuration/options.html#known-local-folder)

File: hivemind/moe/server/connection_handler.py [hunk 4]
```diff
@@ -6,11 +6,12 @@
 import grpc
 import torch
 
-from hivemind.proto import runtime_pb2, runtime_pb2_grpc as runtime_grpc
 from hivemind.moe.server.expert_backend import ExpertBackend
-from hivemind.utils import get_logger, Endpoint, nested_flatten
+from hivemind.proto import runtime_pb2
+from hivemind.proto import runtime_pb2_grpc as runtime_grpc
```
Comment (mryab) [inline hivemind/moe/server/connection_handler.py:11]: Maybe [combine_as_imports](https://pycqa.github.io/isort/docs/configuration/options.html#combine-as-imports) might be helpful in this case

File: tests/test_dht_node.py [hunk 5]
```diff
@@ -9,6 +9,7 @@
 import numpy as np
 import pytest
 from multiaddr import Multiaddr
+from test_utils.dht_swarms import launch_star_shaped_swarm, launch_swarm_in_separate_processes
```
Comment (mryab) [inline tests/test_dht_node.py:12]: test_utils also needs to be recognized as a first-party/local module

**Review by mryab — CHANGES_REQUESTED** [review]
Thank you! Please add 
1) Info about using isort to the style part of the contribution guide
2) Isort as a dependency in requirements-dev
3) Relevant configuration options to avoid mixing third- and first-party modules and splitting "as" statements
4) Rerun isort with this new configuration
After that, we will be able to proceed with merging the PR

File: pyproject.toml [hunk 6]
```diff
@@ -1,3 +1,9 @@
 [tool.black]
 line-length = 119
 required-version = "21.6b0"
+[tool.isort]
```
Comment (mryab) [inline pyproject.toml:4]:
```suggestion
required-version = "21.6b0"

[tool.isort]
```

**Review by mryab — APPROVED** [review]

**Review by justheuristic — APPROVED** [review]

**Comment by justheuristic** [comment]
Perhaps the time has come to re-isort and merge this :)

**Pull request merged**